contracts/contracts/validator-registry/VanillaRegistry.sol

```diff
@@ -46,6 +46,7 @@ contract VanillaRegistry is IVanillaRegistry, VanillaRegistryStorage,
         uint256 _slashingPayoutPeriodBlocks,
         address _owner
     ) external initializer {
+        __Pausable_init();
```

Comment:
I would suggest moving this to a separate change and unblock the CI first. I think this change will need vanilla registry contract upgrade which is on L1. I am not sure how to handle this.

Reply:
This was also causing the CI to fail, need to add it in or the tests compain.

Reply:
I checked with @shaspitz and he Ok'd the change [here](https://primevworkspace.slack.com/archives/C067BA9C7SS/p1737950804989999?thread_ts=1737948734.259909&cid=C067BA9C7SS)